ui/app/serializers/sync/association.js

```diff
@@ -31,4 +32,32 @@ export default class SyncAssociationSerializer extends ApplicationSerializer {
     }
     return payload;
   }
+
+  normalizeFetchByDestinations(payload) {
+    const { store_name, store_type, associated_secrets } = payload.data;
+    const unsynced = [];
+    let lastSync;
```

Comment:
For consistency it might be helpful to call this `lastUpdate`

Reply:
Sure that sounds good I'll change it to `lastUpdated` to match the column header.